integration-tests/test_integration/client-integration.test.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+import { Checkout} from "commerce-sdk";
+import chai, { expect } from "chai";
+import chaiAsPromised from "chai-as-promised";
+import { assert, IsExact } from "conditional-type-checks";
+import { commonParameterPositions } from "@commerce-apps/core";
+
+const BASE_URI =
```

Comment:
We can remove the setting of the base URI for this test.  (And I would love to not set custom baseUri for most default integration tests calls)